article/sections/_7_latent_confounding.tex

```diff
@@ -1,10 +1,10 @@
 \section{Latent Confounding}
```

Comment:
cc @hassan-obeid: Somewhere in this section, perhaps in a footnote, can you link to the project repository and where you perform the various simulations and investigations (i.e., your public notebooks)?